docs/documentation/api-specification/ion-api.md

```diff
@@ -42,7 +41,25 @@ public class SampleExtension extends ExtendM3Trigger {
     this.ion = ion
     this.logger = logger
   }
+```
+
+JsonSlurper method for text parsing:
+```groovy
```

Comment:
This is great, but please include the corresponding JSON as well so it is better understood how the JSON is parsed.